packages/core/src/sass/app/_common/layout/header.scss

```diff
@@ -465,8 +465,8 @@
 .contract-details-header-v2 {
     display: flex;
     justify-content: center;
-    height: var(--core-spacing-2400);
-    padding: var(--core-spacing-600) var(--core-spacing-50);
+    height: var(--core-spacing-2000);
```

Comment:
Please, also check .app-contents--is-dtrader-v2 style, as it's adding margin-top to content under the header based on the header height
![Screenshot 2024-08-20 at 5 17 18 PM](https://github.com/user-attachments/assets/2c5a5e3d-9805-454c-8969-67a664c2e9f7)

Reply:
Plus, in packages/trader/src/AppV2/Containers/Trade/trade.tsx we calculate chart container height, based on header height:
```
const dynamic_chart_height =
        window.innerHeight - HEIGHT.HEADER - HEIGHT.BOTTOM_NAV - HEIGHT.ADVANCED_FOOTER - HEIGHT.PADDING;
```
Will it be possible to add new height in HEIGHT.HEADER?

Reply:
All these changes are needed, because DTrader-V2 header was 48px and old one (which we are going to use) is 40px

Reply:
Sorry, also noticed, that height of bottom nav changed. Was it expected?
![Screenshot 2024-08-21 at 11 42 55 AM](https://github.com/user-attachments/assets/e47b97d8-9fb2-47e2-b4a4-128208104025)
![Screenshot 2024-08-21 at 11 43 19 AM](https://github.com/user-attachments/assets/b6d6d5f8-2710-4780-b2c0-40719628b877)

Reply:
![Uploading Screenshot 2024-08-21 at 11.41.19 AM.png…]()

Reply:
![Screenshot 2024-08-21 at 11 41 19 AM](https://github.com/user-attachments/assets/bf9dad5c-ec88-4e7a-b721-001142001781)